### ISSUE TITLE
Update codecov preferences to be nicer

### ISSUE THREAD

**Issue by laurenmarietta** [spacetelescope/jwql#367]
Codecov is pretty mean. It says we only have 12% test coverage: https://codecov.io/gh/spacetelescope/jwql

Many of the files that are contributing to that low grade are the plots from Daniel's data trending monitor - which really shouldn't be included in this kind of calculation. Also, we will eventually be testing web app code too (#354), so it might be a good idea to include the web app Python scripts in the mix.

Basically, we should see if we can alter the codecov settings to give more helpful feedback on PRs. (Or, maybe not give automatic feedback at all?)

**Comment by laurenmarietta** [comment]
Helpful docs: 
- https://docs.codecov.io/docs/pull-request-comments
- https://docs.codecov.io/docs/ignoring-paths
